### ISSUE TITLE
Exception when running diff command

### ISSUE THREAD

**Issue by samuel-w** [borgbase/vorta#650]
**Describe the bug**
Unhandled exception when running diff. The output of `grep flatpak` is so large that it would be unhelpful, so it's not included.

**To Reproduce**
1. Run a diff between two archives. Mine were made months apart.

**Desktop (please complete the following information):**
- OS: Ubuntu 20.04
- Vorta version: 0.7.1, non-flatpak
- Borg version: 1.1.13

Vorta and Borg versions can be found in Main Window > Misc Tab

**Additional context**
If appropriate include logs. Can be found in Main Window > Misc Tab > Log.
```
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.8/site-packages/vorta/views/archive_tab.py", line 472, in list_diff_result
    window = DiffResult(result['data'], archive_newer, archive_older)
  File "/home/user/.local/lib/python3.8/site-packages/vorta/views/diff_result.py", line 22, in __init__
    files_with_attributes, nested_file_list = parse_diff_lines(fs_data.split('\n'))
  File "/home/user/.local/lib/python3.8/site-packages/vorta/views/diff_result.py", line 94, in parse_diff_lines
    files_with_attributes.append(parse_line(line))
  File "/home/user/.local/lib/python3.8/site-packages/vorta/views/diff_result.py", line 87, in parse_line
    d = get_dict_from_list(nested_file_list, dir.split('/'))
  File "/home/user/.local/lib/python3.8/site-packages/vorta/utils.py", line 57, in get_dict_from_list
    return reduce(operator.getitem, mapList, dataDict)
KeyError: 'flatpak'
```

Output just before crash, has a different traceback, line number is different due to added `print(dataDict, mapList)`:
```
Traceback (most recent call last):
  File "/home/user/Projects/vorta/src/vorta/views/archive_tab.py", line 472, in list_diff_result
    window = DiffResult(result['data'], archive_newer, archive_older)
  File "/home/user/Projects/vorta/src/vorta/views/diff_result.py", line 22, in __init__
    files_with_attributes, nested_file_list = parse_diff_lines(fs_data.split('\n'))
  File "/home/user/Projects/vorta/src/vorta/views/diff_result.py", line 94, in parse_diff_lines
    files_with_attributes.append(parse_line(line))
  File "/home/user/Projects/vorta/src/vorta/views/diff_result.py", line 87, in parse_line
    d = get_dict_from_list(nested_file_list, dir.split('/'))
  File "/home/user/Projects/vorta/src/vorta/utils.py", line 58, in get_dict_from_list
    return reduce(operator.getitem, mapList, dataDict)
KeyError: 'issue'
```
https://gist.githubusercontent.com/samuel-w/9935848475ef3ec1b00a688b004e97c2/raw/f5ba72b49116a7df6ad71b26844aefbf241fe917/Output%2520of%2520crash

**Comment by m3nu** [comment]
Likely related:

```
File "/usr/lib/python3/dist-packages/vorta/views/archive_tab.py", line 486, in list_diff_result
    window = DiffResult(result['data'], archive_newer, archive_older)
  File "/usr/lib/python3/dist-packages/vorta/views/diff_result.py", line 22, in __init__
    files_with_attributes, nested_file_list = parse_diff_lines(fs_data.split('\n'))
  File "/usr/lib/python3/dist-packages/vorta/views/diff_result.py", line 94, in parse_diff_lines
    files_with_attributes.append(parse_line(line))
  File "/usr/lib/python3/dist-packages/vorta/views/diff_result.py", line 87, in parse_line
    d = get_dict_from_list(nested_file_list, dir.split('/'))
  File "/usr/lib/python3/dist-packages/vorta/utils.py", line 112, in get_dict_from_list
    return reduce(operator.getitem, mapList, dataDict)
KeyError: '12'
```

**Comment by m3nu** [comment]
Likely solved by #909
